game/3dobject.cc

```diff
@@ -50,7 +50,7 @@ void Object3d::loadWavefrontObj(fs::path const& filepath, float scale) {
 			texcoords.push_back(glmath::vec2(x, y));
 		} else if (row.substr(0,2) == "vn") {  // Normals
 			srow >> tempst >> x >> y >> z;
-			double sum = std::abs(x)+std::abs(y)+std::abs(z);
+			auto sum = std::abs(x)+std::abs(y)+std::abs(z);
```

Comment:
Not sure about this one, we're converting a sum of three floats into a double, I don't see how it'd lose precision but I'm not really sure why we are converting to a double either.